Package.swift

```diff
@@ -56,6 +57,13 @@ let package = Package(
       name: "SwiftFormatPrettyPrint",
       dependencies: ["SwiftFormatCore", "SwiftFormatConfiguration"]
     ),
+    .target(
+      name: "SwiftFormatWhitespaceLinter",
```

Comment:
I feel like we missed an opportunity to call this `SwiftFormatPrettyLint`.

...but you don't need to change it. Too cutesy.

Reply:
PrettyLinty